gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
```

Comment:
I would prefer to not update the gradle wrapper version in this PR. If we want to it we can do it in a separate PR. 
Also, I will like to let you know that I already started some work to upgrade gradle to much updated version.

Reply:
done